test/blackbox-tests/test-cases/odoc/odoc-wrapped-lib.t/run.t

```diff
@@ -0,0 +1,23 @@
+This test generates documentation for non-hidden modules only for a library:
+
+  $ ocamlc -c -bin-annot foo.ml
```

Comment:
Why do we need this `ocamlc` call?

Reply:
Surely, that’s a redundant call! Good catch!